apps/api/v2/src/modules/organizations/inputs/update-organization-membership.input.ts

```diff
@@ -1,17 +1,19 @@
-import { ApiProperty } from "@nestjs/swagger";
+import { ApiProperty, ApiPropertyOptional } from "@nestjs/swagger";
 import { MembershipRole } from "@prisma/client";
 import { IsBoolean, IsOptional, IsEnum } from "class-validator";
 
 export class UpdateOrgMembershipDto {
   @IsOptional()
   @IsBoolean()
+  @ApiPropertyOptional({ type: Boolean, default: false })
   readonly accepted?: boolean = false;
 
   @IsEnum(MembershipRole)
-  @ApiProperty({ enum: ["MEMBER", "OWNER", "ADMIN"] })
+  @ApiProperty({ enum: ["MEMBER", "OWNER", "ADMIN"], required: true, default: "MEMBER" })
```

Comment:
I think we need `@IsOptional()` here and make it optional property - we are updating membership and role does not have to be defined.